packages/rs-drive/src/drive/document/query/mod.rs

```diff
@@ -199,7 +199,7 @@ impl Drive {
                 platform_version,
             )?
             .ok_or(Error::Query(QuerySyntaxError::DataContractNotFound(
-                "contract not found",
+                "contract not found".to_string(),
```

Comment:
You can use https://docs.rs/thiserror-ext/latest/thiserror_ext/derive.Construct.html so you can pass both Strings and &str to constructors